api/rbdigital.py

```diff
@@ -1410,9 +1410,9 @@ def populate_delta(self, months=1):
         delta = self.get_delta(from_date=(today - time_ago), to_date=today)
         if not delta or len(delta) < 1:
             return None, None
-
-        items_added = delta[0].get("addedTitles", 0)
-        items_removed = delta[0].get("removedTitles", 0)
+        [delta] = delta
+        items_added = delta.get("addedTitles", [])
+        items_removed = delta.get("removedTitles", [])
```

Comment:
This is the bug fix. if `addedTitles` or `removedTitles` are ever missing, we would have ended up calling `len(0)` and crashing.